filter/filterConfig.xml

```diff
@@ -19,12 +19,24 @@
 			description="plugins.importexport.zenodo.description"
 			inputType="class::classes.submission.Submission"
 			outputType="primitive::string" />
+        <!-- Zenodo JSON publication output -->
+        <filterGroup
+            symbolic="publication=>zenodo-json"
+            displayName="plugins.importexport.zenodo.displayName"
+            description="plugins.importexport.zenodo.description"
+            inputType="class::classes.publication.Publication"
+            outputType="primitive::string" />
 	</filterGroups>
 	<filters>
 		<!-- Zenodo JSON article output -->
 		<filter
 			inGroup="article=>zenodo-json"
-			class="APP\plugins\importexport\zenodo\filter\ZenodoJsonFilter"
+			class="APP\plugins\generic\zenodo\filter\ZenodoJsonFilter"
 			isTemplate="0" />
+        <!-- Zenodo JSON publication output -->
+        <filter
+            inGroup="publication=>zenodo-json"
+            class="APP\plugins\generic\zenodo\filter\ZenodoJsonFilter"
+            isTemplate="0" />
```

Comment:
It seems different indents are used in this file -- maybe to make them all the same?